src/components/LogViewer/LogViewerToolbar.tsx

```diff
@@ -24,74 +26,120 @@ export const LogViewerToolbar = memo(
-              onClick={() => {
-                logViewerRef.current?.scrollTo(0, 0);
-                setIsPaused(true);
-              }}
-              variant="control"
-              icon={<LongArrowAltUpIcon />}
-            >
-              Top
-            </Button>
-          </ToolbarItem>
-          <ToolbarItem>
-            <Button
-              onClick={() => {
-                logViewerRef.current?.scrollToBottom();
-                setIsPaused(false);
-              }}
-              variant="control"
-              icon={<LongArrowAltDownIcon />}
-            >
-              Bottom
-            </Button>
-          </ToolbarItem>
-        </ToolbarGroup>
-        <ToolbarGroup align={{ default: 'alignRight' }}>
-          {!isStatic && (
-            <>
-              <ToolbarItem alignSelf="center">
-                <TooltipWrapper tooltip="Scroll down automatically on new log lines input.">
-                  <Checkbox
-                    id="force-following-check"
-                    label="Force following"
-                    isChecked={isFollowing}
-                    onChange={(_, checked) => {
-                      setIsPaused(!checked);
-                      setIsFollowing(checked);
-                    }}
-                  />
-                </TooltipWrapper>
-              </ToolbarItem>
-              <ToolbarItem variant="separator" />
-            </>
-          )}
-          <ToolbarItem alignSelf="center">
-            <Checkbox
-              id="wrap-lines-check"
-              label="Wrap lines"
-              isChecked={areLinesWrapped}
-              onChange={(_, checked) => {
-                setAreLinesWrapped(checked);
-              }}
-            />
-          </ToolbarItem>
-          {!!customActions?.length &&
-            customActions.map((node, index) => (
-              <Fragment key={index}>
+  }: ILogViewerToolbarProps) => {
+    useEffect(() => {
+      const handleKeyDown = (event: KeyboardEvent) => {
+        const target = event.target as HTMLElement;
+        const tagName = target.tagName.toLowerCase();
+        const isInput = tagName === 'input' || tagName === 'textarea' || target.isContentEditable;
+
+        if (isInput) {
+          return;
+        }
+
+        if (event.key === '/') {
+          event.preventDefault();
+          const searchInput = document.querySelector<HTMLInputElement>(
+            '.pf-v5-c-log-viewer__header .pf-v5-c-text-input-group__text-input'
+          );
+          searchInput?.focus();
+        }
+      };
+
+      window.addEventListener('keydown', handleKeyDown);
+      return () => {
+        window.removeEventListener('keydown', handleKeyDown);
+      };
+    }, []);
+
+    useEffect(() => {
+      if (autofocusSearchBar) {
+        const timeoutId = setTimeout(() => {
+          const searchInput = document.querySelector<HTMLInputElement>(
+            '.pf-v5-c-log-viewer__header .pf-v5-c-text-input-group__text-input'
```

Comment:
Did you consider moving searchInput (or some part of it) into the separated constant to prevent code duplication?